.github/workflows/deploy-reusable.yml

```diff
@@ -131,6 +131,9 @@ jobs:
         uses: ./.github/actions/setup-python
         with:
           python_version: 3.9
+      - name: Install Poetry Export Plugin
+        run: |
+          poetry self add poetry-plugin-export
```

Comment:
Poetry v2 was throwing error when serverless was running the command 

```
poetry export --without-hashes -f requirements.txt -o requirements.txt --with-credentials
```

The fix for it is to install the export plugin for poetry in the action.